mailpoet/lib/Automation/Engine/Builder/UpdateAutomationController.php

```diff
@@ -66,6 +66,13 @@ public function updateAutomation(int $id, array $data): Automation {
       }
     }
 
+    $automation->deleteMetas();
```

Comment:
Doesn't this belong inside that IF below? Seems like otherwise it would have the side effects of deleting the meta when anything else is changed.

Reply:
6d6cb32f3